docs/source/usage/picmi/intro.rst

```diff
@@ -113,13 +113,31 @@ Parameters/Methods prefixed with ``picongpu_`` are PIConGPU-exclusive.
 
 - **Simulation**
 
-  - ``__init__(..., picongpu_template_dir)``:
+  additional constructor/configuration options:
+  - ``picongpu_template_dir``:
     Specify the template dir to use for code generation,
     please refer to :ref:`the documentation on the matter for details <picmi-custom-generation>`
-  - ``__init__(...,  picongpu_typical_ppc)`` typical ppc to be used for normalization in PIConGPU
+  - ``picongpu_typical_ppc``:
+    typical ppc to be used for normalization in PIConGPU, if not set explicitly, PIConGPU will use the median ppc of all defined species
+  - ``picongpu_moving_window_move_point``:
+    portion of the simulation window a light ray reaches from the time of the start of the simulation until the simulation window begins to move.
+
+    .. warning::
+
+      If the moving window is active, one gpu in y direction is reserved for initializing new spaces, thereby reducing the simulation window size accordingly
+
+  - ``picongpu_moving_window_stop_iteration``:
+    iteration at which to stop moving the simulation window
+  - ``picongpu_interaction``:
+    ``Interaction`` object specifying all interactions of the simulation, i.e. all ionization models and their configurations and so on.
+    This replaces the PICMI ``add_interaction`` method.
```

Comment:
FMI: Why replace that? Is the PICMI original version so insufficient for our approach?

Reply:
Okay. Found it below. A link to that explanation might be nice. Or put the one unsupported method on top and link the other way round, so the reader is already aware.

Reply:
followed your suggestion